.pre-commit-config.yaml

```diff
@@ -15,7 +15,7 @@ repos:
         stages: [commit]
       - id: check-license
         name: check-license
-        entry: mojo stdlib/scripts/check-licenses.mojo
+        entry: mojo stdlib/scripts/check_licenses.mojo
```

Comment:
Thanks for fixing this - I rolled it into https://github.com/modularml/mojo/pull/3667 which I can merge now to unblock nightly branch.  Note this file isn't managed by copybara.